src/models/informer.py

```diff
@@ -3,3 +3,439 @@
 
 class Informer(TorchModel, config_name='informer'):
     pass
+
+class GaussianBasisFunctions(object):
+    """Function phi(t) = Gaussian(t; mu, sigma_sq)."""
+    def __init__(self, mu, sigma):
+        self.mu = mu.unsqueeze(0)
+        self.sigma = sigma.unsqueeze(0)
+
+    def __repr__(self):
+        return f"GaussianBasisFunction(mu={self.mu}, sigma={self.sigma})"
```

Comment:
Let's use unit quotes be default
```suggestion
        return f'GaussianBasisFunction(mu={self.mu}, sigma={self.sigma})'
```